spec/request/api/v1/user/authentication_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'Authentication /api/v1/users/sign_in', type: :request do
+  before(:each) do
+    @current_user = FactoryBot.create(:user)
```

Comment:
I think here you can use devise helper method `current_user`.

See https://github.com/heartcombo/devise#controller-filters-and-helpers